axelrod/strategies/titfortat.py

```diff
@@ -92,6 +94,39 @@ class TwoTitsForTat(Player):
     def strategy(opponent: Player) -> Action:
         return D if D in opponent.history[-2:] else C
 
+class DynamicTwoTitsForTat(Player):
+    """A player starts by cooperating and then mimics previous move by 
+    opponent with a dynamic bias based off of the opponents ratio of 
+    cooperations to total moves (so their current probability of 
+    cooperating towards cooporating regardless of the move 
+    (aka: forgiveness)."""
```

Comment:
Can you add (at the end of this docstring):

```
Names:

- DynamicTwoTitsForTat: Original name by <your name if you wish>
```

Similar to http://axelrod.readthedocs.io/en/stable/reference/all_strategies.html#axelrod.strategies.ann.EvolvedANN for example.